vertx-sql-client/src/main/java/io/vertx/sqlclient/desc/ColumnDescriptor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2017 Julien Viet
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.vertx.sqlclient.desc;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+import java.sql.JDBCType;
+
+@VertxGen
+public interface ColumnDescriptor {
+
+  String name();
```

Comment:
I don't think it's necessary to have `name()` (or a `ColumnDescriptor` at all for that matter) since the column name is already available through `SqlResult#columnNames()`. Instead, could we just add a `SqlResult#jdbcTypes()` method?